internal/cli/kraft/cloud/certificate/create/create.go

```diff
@@ -34,6 +36,41 @@ type CreateOptions struct {
 	Token  string                             `noattribute:"true"`
 }
 
+func isValidChain(chain string) error {
+	data := []byte(chain)
+	for {
+		block, rest := pem.Decode(data)
+		if block == nil {
+			if len(rest) > 0 {
+				return fmt.Errorf("could not parse PEM")
+			}
+			break
+		}
+		if _, err := x509.ParseCertificates(block.Bytes); err != nil {
+			return fmt.Errorf("could not parse certificate: %w", err)
+		}
+		data = rest
```

Comment:
```diff
- data = rest
```

Reply:
Copy needs to happen to analyze the full chain